lms/static/js/learner_dashboard/views/program_header_view.js

```diff
@@ -0,0 +1,56 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'text!../../../templates/learner_dashboard/program_header_view.underscore',
+            'picturefill'
+           ],
+         function(Backbone, $, pageTpl, picturefill) {
+            return Backbone.View.extend({
+                breakpoints: {
+                    min: {
+                        'medium': '768px',
+                        'large': '1180px',
+                        'x-large': '1440px'
```

Comment:
I'm not sure where to breakpoints for the listing page are, but if they're the same, is there a way they can be shared across views?